tests/conftest.py

```diff
@@ -36,6 +36,15 @@ def pytest_addoption(parser):
     )
 
 
+def setUp():
```

Comment:
`setUp` seems like a pretty generic term.  you're specifically initializing the db static content here - i'd rename to something that's more appropriate.

I don't have a problem with it being a function that the fixtures call, but there needs to be a mechanism to avoid calling it twice, should a test require multiple fixtures, each of which call this function.  Does the fixture mechanism handle this?